app/controllers/rooms_controller.rb

```diff
@@ -0,0 +1,8 @@
+class RoomsController < ApplicationController
+  def index
+    @rooms = Room.all
+    # I was pretty sure that you could call the model method
```

Comment:
You're correct! Referencing the attributes and the model method you created is totally fine and encouraged